src/payment/strategies/paypal-commerce/paypal-commerce-alternative-methods-strategy.ts

```diff
@@ -0,0 +1,145 @@
+import { Cart } from '../../../cart';
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { InvalidArgumentError } from '../../../common/error/errors';
+import { OrderActionCreator, OrderRequestBody } from '../../../order';
+import { OrderFinalizationNotRequiredError } from '../../../order/errors';
+import { PaymentArgumentInvalidError, PaymentMethodInvalidError } from '../../errors';
+import PaymentActionCreator from '../../payment-action-creator';
+import PaymentMethodActionCreator from '../../payment-method-action-creator';
+import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
+import PaymentStrategy from '../payment-strategy';
+
+import { ApproveDataOptions,
+    ButtonsOptions,
+    PaypalCommerceCreditCardPaymentInitializeOptions,
+    PaypalCommerceInitializationData,
+    PaypalCommercePaymentInitializeOptions,
+    PaypalCommercePaymentProcessor, PaypalCommerceScriptParams,
+    PaypalCommerceSDKFunding } from './index';
```

Comment:
We don't import from `./index` as it'd be a recursive import. (`index` also points to this file.)

Reply:
removed file as it is no longer needed